src/service/platform-impl.js

```diff
@@ -136,6 +136,25 @@ export class Platform {
     }
     return parseInt(res[index], 10);
   }
+
+  /**
+   * Returns the minor ios version in string.
+   * The ios version can contain two numbers (10.2) or three numbers (10.2.1).
+   * Direct string equality check is not suggested, use startWith instead.
+   * @returns {string}
+   */
+  getIosVersion() {
+    if (!this.navigator_.userAgent) {
+      return '';
+    }
+    let version = this.navigator_.userAgent
```

Comment:
Sure. But there could be "OS ???" in other browsers as well. Since the method is called `getIosVersionString`, we should make sure it's indeed iOS.